### DISCUSSION TITLE
User not found

### DISCUSSION THREAD

**Discussion by mremptypants** [RocketChat/Rocket.Chat#34031] [Q&A]
Moved to 7.0.0 Docker, this is a private small server but the older users can not sign in again, the old connections are fine just do not sign out.  New users can make accounts and sign in and out fine, so the database seems ok. I can not do anything in the admin portal being it is not linking to my account right.  

Anyone point me in the way to fix the database or would we all have to remake accounts?  I have rebuilt the both dockers from the ground up and restored the old database, i guessed it would not work being it is the same data but I wanted to try.

Really at a loss here really. 
![image](https://github.com/user-attachments/assets/40953dcb-4689-4ece-a48f-bfe84a8534e3)

Any ideas or help would be great

**Comment by reetp** [comment]
Did you restore from a backup?

The correct places to ask these questions are here:

https://open.rocket.chat/channel/support/
https://forums.rocket.chat/
